src/Tizen.TV.UIControls.Forms/GridView.cs

```diff
@@ -0,0 +1,235 @@
+
+        /// <summary>
+        /// Identifies the SelectedItem bindable property.
+        /// </summary>
+        public static readonly BindableProperty SelectedItemProperty = BindableProperty.Create(nameof(SelectedItem), typeof(object), typeof(GridView), null, propertyChanged: (b, o, n) => ((GridView)b).UpdateSelectedItems(), defaultBindingMode: BindingMode.TwoWay);
+
+        /// <summary>
+        /// Identifies the ItemHorizontalAlignment bindable property.
+        /// </summary>
+        public static readonly BindableProperty ItemHorizontalAlignmentProperty = BindableProperty.Create("ItemHorizontalAlignment", typeof(LayoutAlignment), typeof(GridView), LayoutAlignment.Center);
+
+        /// <summary>
+        /// Identifies the ItemVerticalAlignment bindable property.
+        /// </summary>
+        public static readonly BindableProperty ItemVerticalAlignmentProperty = BindableProperty.Create("ItemVerticalAlignment", typeof(LayoutAlignment), typeof(GridView), LayoutAlignment.Center);
+
+        /// <summary>
+        /// ItemSelected is raised when one item of GridView is selected.
+        /// </summary>
+        public event EventHandler<SelectedItemChangedEventArgs> ItemSelected;
+
+        /// <summary>
+        /// ItemFocused is raised when one item of GridView has received focus.
+        /// </summary>
+        public event EventHandler<GridViewFocusedEventArgs> ItemFocused;
+
+        /// <summary>
+        /// Gets or sets the source of items to template and display.
+        /// </summary>
+        public IEnumerable ItemsSource
+        {
+            get => (IEnumerable)GetValue(ItemsSourceProperty);
+            set => SetValue(ItemsSourceProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets the DataTemplate to apply to the ItemsSource.
+        /// </summary>
+        public DataTemplate ItemTemplate
+        {
+            get => (DataTemplate)GetValue(ItemTemplateProperty);
+            set => SetValue(ItemTemplateProperty, value);
+        }
+
+        /// <summary>
+        /// Gets or sets a value that represents the height of an item.
+        /// </summary>
+        public double ItemHeight
+        {
+            get { return (double)GetValue(ItemHeightProperty); }
+            set { SetValue(ItemHeightProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value that represents the width of an item.
+        /// </summary>
+        public double ItemWidth
+        {
+            get { return (double)GetValue(ItemWidthProperty); }
+            set { SetValue(ItemWidthProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the item's alignment along X-axis within a GridView.
+        /// Accepted values range from 0.0 to 1.0 range.
```

Comment:
I think it's an incorrect description. Please update it.

Reply:
Thank you for your recommendation.